js/ui/draggable.js

```diff
@@ -461,12 +465,15 @@ const Draggable = DOMComponent.inherit({
         if(itemsSelector[0] === '>') {
             itemsSelector = itemsSelector.slice(1);
         }
-
         eventsEngine.on($element, DRAGSTART_EVENT_NAME, itemsSelector, data, this._dragStartHandler.bind(this));
         eventsEngine.on($element, DRAG_EVENT_NAME, data, this._dragMoveHandler.bind(this));
         eventsEngine.on($element, DRAGEND_EVENT_NAME, data, this._dragEndHandler.bind(this));
         eventsEngine.on($element, DRAG_ENTER_EVENT_NAME, data, this._dragEnterHandler.bind(this));
         eventsEngine.on($element, DRAGEND_LEAVE_EVENT_NAME, data, this._dragLeaveHandler.bind(this));
+
+        if(this.option('onCancelByEsc')) {
+            eventsEngine.on($element, KEYDOWN_EVENT_NAME, this._escapeHandler.bind(this));
```

Comment:
Let's rename the handler to ```_keydownHandler```, because it may handle other keys in the future :)

```suggestion
            eventsEngine.on($element, KEYDOWN_EVENT_NAME, this._keydownHandler.bind(this));
```